docs/syntaxes/reason.json

```diff
@@ -1279,7 +1279,7 @@
         {
           "match": "\\b([[:lower:]][[:word:]]*)\\b(?!\\.[[:upper:]])",
           "captures": {
-            "1": { "name": "variable.language string.other.link" }
+            "1": { "name": "variable.language" }
```

Comment:
`pattern-variable` is the only one needed to fix the reported issue. But I removed other appearances as well because i can't think of any cases where we'd like the code to appear underlined.